src/components/grid/GridComponent.vue

```diff
@@ -128,31 +152,72 @@ export default Vue.extend({
     rsql: function () {
       this.loadGridData()
     }
-  },
-  created () {
-    this.loadAssessments()
-    this.loadGridData()
   }
 })
 </script>
 
 <style scoped>
+  table {
+    overflow: hidden;
+    position: relative;
+  }
+  table th:first-child {
+    width: 15rem;
+    max-width: 15rem;
+    overflow: hidden;
+  }
+  table td,
+  table th:not(:first-child){
+    width: 4rem;
+    max-width: 4rem;
+    min-width: 4rem;
+  }
   table th {
     white-space: nowrap;
     vertical-align: middle;
     font-weight: normal;
   }
+  .sticky{
+    position: fixed;
+    top:0;
+    background-color: white;
+    background: linear-gradient(180deg, rgba(255,255,255,1) 0%, rgba(255,255,255,0.9) 75%, rgba(255,255,255,0) 100%);
+    z-index: 2000;
+  }
+  .sticky .assessments-title{
+    height: 8rem;
+  }
+  .sticky .assessments-title span{
+    bottom: 1rem;
+  }
+  .space-holder{
+    height: 96px;
```

Comment:
why 96 px ?

Reply:
Needs to be the same size as `.assessments-title`. Changed to the correct `6rem`